charts/grafana/templates/_helpers.tpl

```diff
@@ -113,4 +113,16 @@ Return the appropriate apiVersion for rbac.
 {{- else -}}
 {{- print "rbac.authorization.k8s.io/v1beta1" -}}
 {{- end -}}
```

Comment:
This is the reason for the lint errors `unexpected <define> in command`. The previous define is missing its final end block.

```suggestion
{{- end -}}
{{- end -}}
```